ReSharper.FSharp/src/FSharp.Psi.Daemon/src/Analyzers/LambdaAnalyzer.fs

```diff
@@ -84,45 +85,83 @@ type LambdaAnalyzer() =
 
         compareArgsRec expr 0 null
 
-    let isImplicitlyConvertedToDelegate (lambda: ILambdaExpr) =
+    let canBeNamedArg (expr: IBinaryAppExpr) =
+        if isNull expr.Operator then false else
+
+        let exprReference = expr.Operator.Reference
+        if exprReference.GetName() <> "=" then false else
```

Comment:
I wonder if we could add `ShortName` to `IBinaryAppExpr`, similar to how it's added to `IReferenceExpr`? It'll hide all the `null` checks.

Reply:
i think we can do it in a separate PR